src/dispatch/plugins/dispatch_slack/menus.py

```diff
@@ -39,7 +42,9 @@ async def handle_slack_menu(*, config, client: WebClient, request: Request, orga
         db_session = get_organization_scope_from_slug(organization)
 
     f = menu_functions(action_id)
-    return f(db_session, user_id, user_email, channel_id, incident_id, query_str, request)
+    menu = f(db_session, user_id, user_email, channel_id, incident_id, query_str, request)
+    db_session.close()
```

Comment:
Should we close the session if `f` needs it?

Reply:
It's only closed after f returns, so it would be able to use it.

Reply:
Ah, never mind. Read the code too fast.